### PR TITLE
Update TR Locales

### PR DESCRIPTION
**Pull request by mikropsoft**
Also the number of lines is equalised with English

### CODE REVIEW

File: app/core/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -164,81 +165,83 @@
...
-    <string name="post_notifications_denied">Bu işlevi etkinleştirmek için bildirim izni veriniz.</string>
-    <string name="install_unknown_denied">Bu işlevi etkinleştirmek için "Bilinmeyen uygulamaları yükle" ayarına izin veriniz.</string>
+    <string name="external_rw_permission_denied">Bu işlevselliği etkinleştirmek için depolama izni verin</string>
+    <string name="post_notifications_denied">Bu işlevselliği etkinleştirmek için bildirim izni verin</string>
+    <string name="install_unknown_denied">Bu işlevselliği etkinleştirmek için "bilinmeyen uygulamaları yükle" iznini verin</string>
     <string name="add_shortcut_title">Ana ekrana kısayol ekle</string>
-    <string name="add_shortcut_msg">Bu uygulamayı gizledikten sonra adını ve simgesini tanımak zorlaşabilir. Ana ekrana güzel bir kısayol eklemek ister misiniz?</string>
-    <string name="app_not_found">Bu eylemi gerçekleştirecek uygulama bulunamadı</string>
+    <string name="add_shortcut_msg">Bu uygulamayı gizledikten sonra adı ve simgesi tanınmayabilir. Ana ekrana güzel bir kısayol eklemek ister misiniz?</string>
```
Comment (yujincheng08) [inline app/core/src/main/res/values-tr/strings.xml:242]: missing `>`

**Comment by yujincheng08** [comment]
cannot build

**Comment by mikropsoft** [comment]
> cannot build

I don't see any problem, this is weird

**Comment by topjohnwu** [comment]
`Magisk/app/core/src/main/res/values-tr/strings.xml:247:3: Resource and asset merger: The element type "string" must be terminated by the matching end-tag "</string>".`

**Comment by mikropsoft** [comment]
> `Magisk/app/core/src/main/res/values-tr/strings.xml:247:3: Resource and asset merger: The element type "string" must be terminated by the matching end-tag "</string>".`

This already is. This change is not possible even in Commit.

**Comment by mikropsoft** [comment]
> `Magisk/app/core/src/main/res/values-tr/strings.xml:247:3: Resource and asset merger: The element type "string" must be terminated by the matching end-tag "</string>".`

Is it ready now?

**Review by topjohnwu — APPROVED** [review]

**Pull request merged**